website/blog/2020-10-29-presto-at-drift.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Presto Enables Internal Log Data Analysis at Drift
+author: Arun Venkateswaran
+authorURL: https://github.com/venkaa6
+---
+
+I’m a Senior Software Engineer in the data group at Drift, a conversational marketing platform that is used for qualifying leads faster, automatically booking meetings and connecting customers to the right business solutions more efficiently. I’ve used Presto quite a bit throughout my career, and I want to first give readers a quick overview of how Presto has enabled my team at Drift to quickly and cost-effectively analyze distributed logs at scale. Then I will share how we used and benefited from Presto at Vistaprint, where I worked previously.
+
```

Comment:
Add a truncate tag after the first paragraph so that the entire blog post doesn't appear in the index.
```suggestion
<!--truncate-->
```